feature/validate-subject-pool/src/main/res/layout/fragment_validate_subject_pool.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We've previously had issues with the horizontal orientation when using the `layout_constraint*_bias` and `layout_constraint*_percent` params. Can you validate that this layout looks properly when the mobile device is rotated?

Reply:
I checked both orientations in the preview. The layout is very basic so I think it should be fine.